packages/lesswrong/components/recentDiscussion/RecentDiscussionThread.tsx

```diff
@@ -187,17 +212,34 @@ const RecentDiscussionThread = ({
 
   return (
     <AnalyticsContext pageSubSectionContext='recentDiscussionThread'>
-      <div className={classes.root}>
-        <div className={classes.post}>
+      <div className={classNames(
+        classes.root,
+        {
+          [classes.plainBackground]: !subforumIntroPost,
+          [classes.primaryBackground]: subforumIntroPost
+        }
+      )}>
+        <div className={classNames(
+          classes.post,
+          {
+            [classes.plainBackground]: !subforumIntroPost,
+            [classes.primaryBackground]: subforumIntroPost
+          }
+        )}>
```

Comment:
Do we need to add a `background-color` here? I think just adding it to the parent `div` is enough

Reply:
Without it it looks like this:

![Screenshot 2023-01-18 at 10 56 51](https://user-images.githubusercontent.com/77623106/213154026-6ed6340f-0748-43b3-81c2-b65d14500ea1.png)

I'm sure there's a way to refactor it so it only needs the outer one but I'm going to leave it